packages/vscode-extension/src/commonlib/azureLogin.ts

```diff
@@ -565,6 +596,19 @@ export class AzureAccountManager extends login implements AzureAccountProvider {
       return undefined;
     }
   }
+
+  checkVersion(): boolean {
+    try {
+      const version: string =
+        vscode.extensions.getExtension<AzureAccount>("ms-vscode.azure-account")!.packageJSON
+          .version;
+      const versionDetail = version.split(".");
+      return parseInt(versionDetail[1]) < 10;
```

Comment:
To be safe, check major version as well... In case it publishes `1.x.x`

Reply:
Thanks for the advice, fixed.